WcaOnRails/app/models/user.rb

```diff
@@ -293,6 +285,44 @@ def after_confirmation
     end
   end
 
+  def software_team?
+    team_member?('software') != false
```

Comment:
I have changed this to have all methods return true or false.
This means that the team_leader? code got longer...